### PR TITLE
Updated about page to include information on Private Repositories

### PR DESCRIPTION
**Pull request by IanLee1521**
Looking for feedback on this change to the about page to add information about Private GitHub repositories that we (Comp) will be making available for the organization. @tgamblin, this is the draft that we were talking about.

cc/ @schulzm @trujillo13 @lee218llnl @schulzm

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: About the LLNL Software Portal
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+Welcome to the Lawrence Livermore National Laboratory software portal! The purpose of this software portal (or catalog if you prefer) is to serve as the definitive source of truth pointing to all software that has been produced and released by Lawrence Livermore National Laboratory.
```
Comment (tgamblin) [inline about/index.md]: delete "or catalog if you prefer"

Comment (tgamblin) [inline about/index.md]: "definitive source of truth" and "all software that has been produced and released" are inaccurate.  I think you should tone this part down.  The portal is for open source software (we release stuff that's not open source too), and "definitive source of truth" sounds overly strong.  Why not "This software portal is LLNL's official hub for open source software" or something similar?

File: about/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+LLNL produces software on a daily basis. Some of this software is used only internally, other components are licensed for use by external partners and collaborators, still other software is released, or even actively developed, in the open on software hosting platforms such as GitHub.com, Bitbucket.org, Sourceforge.net, and others.
+
+This page is designed to be a single stop to find all types of software produced and available due to the expertise of Lawrence Livermore National Laboratory scientists and engineers.
```
Comment (tgamblin) [inline about/index.md]: This sentence seems redundant when taken with paragraph 1.  Do we have some other language we could use that highlights LLNL's enthusiasm for OSS?

File: about/index.md (hunk 1 above)
Comment (IanLee1521) [inline about/index.md]: Longer term my intention was for this to cover both open source and other software we release. But I think that at least in the early phases, while it's mainly focused on open source, I'll update the wording to reflect that.

**Comment by IanLee1521** [comment]
Updated with feedback from @tgamblin. I also took out the line that was redundant. 

In particular though, does the wording about the use of private repositories seem reasonable?

**Comment by tgamblin** [comment]
Looks good to me!

**Pull request merged**